mesheryctl/internal/cli/root/system/reset.go

```diff
@@ -39,6 +41,29 @@ var resetCmd = &cobra.Command{
 // resets meshery config
 func resetMesheryConfig() error {
 	log.Info("Meshery resetting...")
+
+	mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
+	if err != nil {
+		return errors.Wrap(err, "error processing config")
+	}
+
+	currentContext := mctlCfg.CurrentContext
+	currChannel := mctlCfg.Contexts[currentContext].Channel
```

Comment:
The `system` level struct should be filtering any invalid values for channel and version, so confirmation of valid values shouldn't have to be considered here.

@anirudhjain75 will you confirm that this is the case?